sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIOTest.java

```diff
@@ -401,412 +380,412 @@ public void testReadingData() {
    * HadoopInputFormatIO.HadoopInputFormatBoundedSource#populateDisplayData()
    * populateDisplayData()}.
    */
-  @Test
-  public void testReadDisplayData() {
-    HadoopInputFormatBoundedSource<Text, Employee> boundedSource =
-        new HadoopInputFormatBoundedSource<>(
-            serConf,
-            WritableCoder.of(Text.class),
-            AvroCoder.of(Employee.class),
-            null, // No key translation required.
-            null, // No value translation required.
-            new SerializableSplit());
-    DisplayData displayData = DisplayData.from(boundedSource);
-    assertThat(
-        displayData,
-        hasDisplayItem(
-            "mapreduce.job.inputformat.class",
-            serConf.get().get("mapreduce.job.inputformat.class")));
-    assertThat(displayData, hasDisplayItem("key.class", serConf.get().get("key.class")));
-    assertThat(displayData, hasDisplayItem("value.class", serConf.get().get("value.class")));
-  }
+  //  @Test
```

Comment:
Actually, we have to remove all these commented tests there because they tested internal functionality of `HadoopInputFormatIO` which has been moved to `HadoopFormatIO`. So, all these tests were moved into `HadoopFormatIOReadTest`. 
Now I left the only tests in `HadoopInputFormatIOTest` that do testing of public API which still can be used by user.
I forgot to delete these tests before. Thank you for pointing this out.